lightly_studio_view/src/lib/components/SampleDetails/SampleDetailsSidePanel/SampleDetailsSidePanel.svelte

```diff
@@ -51,6 +51,25 @@
     const { isEditingMode } = page.data.globalStorage;
     const annotationLabels = useAnnotationLabels();
     const items = $derived(getSelectionItems($annotationLabels.data || []));
+
+    // Auto-scroll to selected annotation
+    $effect(() => {
+        if (selectedAnnotationId) {
+            // Use requestAnimationFrame to ensure DOM is fully updated
+            requestAnimationFrame(() => {
```

Comment:
`$effect` runs after the component has been mounted on the DOM, so you don't need to use `requestAnimationFrame`.